test/resources/microshift-config.resource

```diff
@@ -10,62 +10,23 @@ Resource            common.resource
 
 
 *** Keywords ***
-Save Default MicroShift Config
```

Comment:
What if you want to save the current configuration because you want to replace entire bits of it? You would need this keyword (and others too) to achieve that, as drop in is an aggregate view and cant delete anything, right? (unless setting null values to parameters, but thats convoluted)

Reply:
Maybe, but that hasn't been the case in our tests. We start VMs with:
```
apiServer:
  subjectAltNames:
  - item1
  - item2
```

And array are not being summed together, only overwritten.
There are two tests that overwrite the `subjectAltNames`: `multi-nic.robot` and `validate-custom-certificates.robot` and using drop-in with the SANs totally works.

Reply:
Un-deleted the keywords. Let's keep them for time being